### DISCUSSION TITLE
.eslintrc.* file name conventions

### DISCUSSION THREAD

**Discussion by dysphasia** [eslint/eslint#15406] [General]
Hi all,

Is there a reason why the `.eslintrc.*` uses the dot file notation? It's not typically a file thats ignored by version control and is a file that should be seen and edited by users, so having it be hidden to a filesystem seems counter-intuitive. I assume there is a good reason, just curious what it is.

Thanks

**Comment by dysphasia** [comment]
After some more digging it looks like this is addressed in https://github.com/eslint/rfcs/pull/9 and https://github.com/eslint/eslint/tree/simpleconfig. Is this planned for release in an upcoming version?

**Reply by nzakas** [reply]
You can follow the progress here:
https://github.com/eslint/eslint/issues/13481

We don’t target release versions. Versions are determined by which features are ready to ship.
